### PR TITLE
Moving credentials into secrets

### PR DESCRIPTION
**Pull request by jupierce**
@bparees 

Also removing admin credentials which do not seem to be used.

### CODE REVIEW

File: openshift/templates/dancer-mysql.json [hunk 1]
```diff
@@ -115,10 +126,7 @@
               "secret": "${GITHUB_WEBHOOK_SECRET}"
             }
           }
-        ],
-        "postCommit": {
-          "script": "perl -I extlib/lib/perl5 -I lib t/*"
-        }
```
Comment (bparees) [inline openshift/templates/dancer-mysql.json:121]: why is this being removed?

Comment (jupierce) [reply openshift/templates/dancer-mysql.json:121]: For other repos, the origin copies seemed to be newer (e.g. the inclusion of the "message" field in the template), so I used that as a heuristic for all updates. Looks like it was wrong for this file. I will restore it.

**Comment by jupierce** [comment]
@bparees ptal

**Comment by bparees** [comment]
lgtm

**Pull request merged**